pkg/apis/servicecatalog/validation/binding_test.go

```diff
@@ -564,6 +564,15 @@ func TestValidateServiceBinding(t *testing.T) {
 			}(),
 			valid: true,
 		},
+		{
+			name: "empty unbind status on update",
+			binding: func() *servicecatalog.ServiceBinding {
+				b := validServiceBinding()
+				b.Status.UnbindStatus = ""
+				return b
+			}(),
+			valid: false,
```

Comment:
Failed when set the `valid: true`, see below, but as @pmorie mentiond in [bug 1566328](https://bugzilla.redhat.com/show_bug.cgi?id=1566328#c5), there should be a conversion that sets the field value to the right value (NotRequired) if it is unset. So, how to add this conversion in e2e test? I could NOT find the conversion, maybe I misunderstand something. 
```go
[root@localhost validation]# go test -v -test.run TestValidateServiceBinding
=== RUN   TestValidateServiceBinding
--- PASS: TestValidateServiceBinding (0.00s)
PASS
ok  	github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/validation	0.008s
[root@localhost validation]# vim binding_test.go 
[root@localhost validation]# go test -v -test.run TestValidateServiceBinding
=== RUN   TestValidateServiceBinding
--- FAIL: TestValidateServiceBinding (0.00s)
	binding_test.go:582: empty unbind status on update: unexpected error: [status.unbindStatus: Unsupported value: "": supported values: "Required", "Succeeded", "Failed", "NotRequired"]
FAIL
exit status 1
FAIL	github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/validation	0.007s
```

Reply:
Not sure where the conversion could be done, but if it's not done by validation, the next best place would be an integration test. From there you can work backwards to see if it gets set somewhere that is testable by an integration test.

Reply:
looks like it should hit something in this block
https://github.com/jianzhangbjz/service-catalog/blob/1e72d2cde11bfea2ccbf42a53ca52daee87f7ea9/pkg/apis/servicecatalog/validation/binding.go#L151-L159

Reply:
Yes, you have the right output. Working as designed here, maybe a good test to add.

Reply:
https://github.com/kubernetes-incubator/service-catalog/issues/2185 created as explanation / follow up.